content/docs/concepts/security/index.md

```diff
@@ -520,12 +520,13 @@ objects, `RbacConfig` is defined as a
 In the `RbacConfig` object, the operator can specify a `mode` value, which can
 be:
 
-* **OFF**: Istio authorization is disabled.
-* **ON**: Istio authorization is enabled for all services in the mesh.
-* **ON_WITH_INCLUSION**: Istio authorization is enabled only for services and
+* **`OFF`**: Istio authorization is disabled.
```

Comment:
please use the markdown `-` instead of `*`.  They produce the same output, however, its helpful for the website to have a consistent implementation.